webapp/src/components/EstateDetailPage/EstateDetailPage.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 
 import Estate from 'components/Estate'
 import { parcelType, publicationType } from 'components/types'
-import EstateDetail from './EstateDetail'
+import EstateDetail from 'components/EstateDetailPage/EstateDetail'
```

Comment:
We tend to keep the relative imports for components on the same folder, but you don't really need to change this

Reply:
God 🤒